Cannot make sandstorm on clean VM

```
Finding meteor-tool installation (can take a few seconds)... meteor-tool@1.3.1
cd icons && /home/user/.meteor/packages/meteor-tool/.1.3.1.1ult9zk++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/bin/npm install
/usr/bin/env: node: No such file or directory
make: *** [icons/node_modules] Error 127
```

So Meteor is installed, it finds `npm`, but not `node`. I suspect `node` is not added to `PATH`?

--- a/Makefile
+++ b/Makefile
@@ -253,7 +253,7 @@ tmp/.shell-env: tmp/.ekam-run $(IMAGES) shell/client/changelog.html shell/client
 	@cd shell/ && meteor npm install
 
 icons/node_modules: icons/package.json
-	cd icons && $(METEOR_DEV_BUNDLE)/bin/npm install
+	cd icons && PATH=$(METEOR_DEV_BUNDLE)/bin:$$PATH $(METEOR_DEV_BUNDLE)/bin/npm install
 
 shell/client/styles/_icons.scss: icons/node_modules icons/*svg icons/Gruntfile.js
 	cd icons && PATH=$(METEOR_DEV_BUNDLE)/bin:$$PATH ./node_modules/.bin/grunt

